bigcache.go

```diff
@@ -12,6 +13,12 @@ const (
 	minimumEntriesInShard = 10 // Minimum number of entries in single shard
 )
 
+// ErrInvalidIteratorState is reported when iterator is in invalid state and cannot be used.
+var ErrInvalidIteratorState = errors.New("Iterator is in invalid state. Use SetNext() to move to next position.")
+
+// ErrCannotRetrieveEntry is reported when entry cannot be retrieved from underlying
+var ErrCannotRetrieveEntry = errors.New("Could not retrieve entry from cache")
+
```

Comment:
This two should not be public. Use empty struct withe Error method like we do in [entry_not_found_error.go](https://github.com/allegro/bigcache/blob/master/entry_not_found_error.go) or make this private. Take a look at http://dave.cheney.net/2016/04/07/constant-errors